packages/webapp/src/components/Animals/SexDetails/SexDetailsCountInput/index.tsx

```diff
@@ -37,6 +37,7 @@ export default function SexDetailsCountInput({
     initialValue: initialCount,
     max,
     clampOnBlur: false,
+    useGrouping: false,
```

Comment:
This disables the thousands grouping with comma or dot, as was discussed in grooming.